src/sasctl/pzmm/zipModel.py

```diff
@@ -4,12 +4,13 @@
 # %%
 from pathlib import Path
 import zipfile
+import io
 
 # %%
 
 class ZipModel():
     
-    def zipFiles(self, fileDir, modelPrefix):
+    def zipFiles(fileDir, modelPrefix):
```

Comment:
def zipFiles(self, fileDir, modelPrefix):

Reply:
Fixed in [07ad6ce](https://github.com/sassoftware/python-sasctl/pull/76/commits/07ad6ce2532e24b1ef56e54c5b7ec572621f1fb0)